### ISSUE TITLE
Menu: accessibility fix

### ISSUE THREAD

**Issue by PosikBoy** [gravity-ui/uikit#2500]
### Menu

The component currently renders links (<a>) with the role menuitem inside ```<li>``` elements.
According to the WAI-ARIA specification, menuitem elements should be direct children of an element with the role menu or menubar, without additional structural elements in between.

WAI-ARIA reference:
https://www.w3.org/WAI/ARIA/apg/patterns/menubar/#wai-ariaroles,states,andproperties

Having ```<li>```between menu and menuitem violates the ARIA hierarchy and may cause incorrect behavior in screen readers.

I suggest adding:

```<li role="presentation">```

This follows the MDN recommendation for cases where li should not participate in the accessibility tree:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/menuitem_role

### Reproduction

https://stackblitz.com/edit/gravity-ui-uikit-example-a6g8mjpn?file=src%2FApp.tsx

### Environment

MacOS, Yandex Browser

### Validations

- [x] Check that there isn't [already an issue](https://github.com/gravity-ui/uikit/issues) that reports the same bug to avoid creating a duplicate.
- [x] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.
